.gitignore

```diff
@@ -17,6 +17,9 @@ bin
 
 !vendor/**/zz_generated.*
 
+# go vendor
+vendor/
```

Comment:
Why remove the vendor directory？This may make it very difficult to compile kruise in a new environment.